.env.example

```
@@ -18,6 +18,8 @@ AUTH_SECRET=""
 # Next Auth Discord Provider
 AUTH_DISCORD_ID=""
 AUTH_DISCORD_SECRET=""
+AUTH_GITHUB_ID=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)